hyperdrive_test.go

```diff
@@ -25,7 +25,9 @@ import (
 )
 
 func init() {
-	mrand.Seed(time.Now().Unix())
+	seed := time.Now().Unix()
+	log.Printf("seed = %v", seed)
```

Comment:
Why do we print the seed?

Reply:
If the test fails, we can use the seed and reproduce the failure more easily.